### PR TITLE
[luci] Replace Argmax quantize verification test

### PR DESCRIPTION
**Pull request by dayo09**
This commit replaces argmax quantize verification tests.
Specifically, it tests with wrong type not with dimension but with input node, with more test variant.
And it tests with wrong granularity with more test variant.
The removed wrong type test with dimension node will be revived after the implementation of proper test macros.

ONE-DCO-1.0-Signed-off-by: Dayoung Lee <dayg502@gmail.com>

----

For #6635

### CODE REVIEW

File: compiler/luci/pass/src/QuantizedModelVerifier.test.cpp [hunk 1]
```diff
@@ -502,7 +502,7 @@ template <Type indexT> class ArgMaxTestGraph final : public SimpleTestGraph
     set_minmax_to_non_const(g(), -1, 1);
   }
 
-public:
+private:
```
Comment (seanshpark) [inline compiler/luci/pass/src/QuantizedModelVerifier.test.cpp:505]: I think this change can be removed from this PR.

File: compiler/luci/pass/src/QuantizedModelVerifier.test.cpp [hunk 2]
```diff
@@ -1141,18 +1141,30 @@ TEST(QuantizedModelVerifierTest, ArgMax_wrong_dimension_type_NEG)
   EXPECT_ANY_THROW(verifier.verify(g.g()));
 }
 
-TEST(QuantizedModelVerifierTest, ArgMax_wrong_input_granularity_NEG)
+TEST(QuantizedModelVerifierTest, ArgMax_wrong_dimension_type_NEG)
```
Comment (seanshpark) [inline compiler/luci/pass/src/QuantizedModelVerifier.test.cpp:1144]: there's same `ArgMax_wrong_dimension_type_NEG` above

**Comment by seanshpark** [comment]
@nnfw-bot test nncc-debug

**Comment by dayo09** [comment]
@seanshpark Test done :-D

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**